### ISSUE TITLE
Guess the number

### ISSUE THREAD

**Issue by jigar-sable** [prathimacode-hub/Awesome_Python_Scripts#61]
---
Title: Guess the number
About: Store a random number in a variable and make the user guess the correct number.
Name: Jigar
Label: Feature Request
Assignee: ''

---

Define You:

- [x] LGM-SOC'21 Participant

**Describe the solution you'd like...**
I'll use random library to generate a random number within the range of 0 to 100.
And then store it in a variable and iterate it via while loop till the correct number is entered by the user and will use if else statement to show if the number is higher or lower then the number entered.

**Comment by prathimacode-hub** [comment]
Interesting game. Awaiting your decision to choose 1 of the 2 issues raised to assign. @jigar-sable

**Comment by jigar-sable** [comment]
> Interesting game. Awaiting your decision to choose 1 of the 2 issues raised to assign. @jigar-sable

Thank you ma'am I'll work on this issue first and then for Phone Number Info #64

**Comment by prathimacode-hub** [comment]
Sure. Issue is assigned to you. @jigar-sable

**Comment by jigar-sable** [comment]
> Sure. Issue is assigned to you. @jigar-sable

Thanks ma'am will start working on it..

**Comment by prathimacode-hub** [comment]
Yes, it's a game. It clearly goes into "PyGamesScripts". @jigar-sable

**Comment by jigar-sable** [comment]
Alright ma'am thank you!!
